ark/utils/data_utils.py

```diff
@@ -205,6 +209,9 @@ def generate_pixel_cluster_mask(fovs, base_dir, seg_dir, pixel_consensus_dir,
             os.path.join(base_dir, pixel_consensus_dir, fov + '.feather')
         )
 
+        # ensure integer display and not float
+        fov_data[cluster_col] = fov_data[cluster_col].astype(int)
+
         # read the segmentation mask to determine size of pixel cluster mask
         seg_mask = io.imread(os.path.join(seg_dir, fov + '_feature_0.tif'))
```

Comment:
I was testing this with another dataset generated by Mesmer, and reading in seg_mask returned a (1,1024,1024) array (the TB example dataset returned (1024,1024)). This caused problems at this point. Maybe a np.squeeze function would be helpful here.

Reply:
Sure, I'll add that in.